src/main/java/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java

```diff
@@ -984,21 +974,16 @@ private void writeStreamResponse(ThreadLocal<JsonGenerator> outputJsonGen,
     ForkJoinPool threadPool = new ForkJoinPool(ProcessingData.getNumberOfDataExtractionThreads());
     try {
       threadPool.submit(() -> stream.parallel().map(data -> {
-        // 0. resolve tags
-        Map<String, Object> tags = data.getProperties();
-        List<String> keysToDelete = new LinkedList<>();
-        List<OSMTag> tagsToAdd = new LinkedList<>();
-        for (Entry<String, Object> tag : tags.entrySet()) {
-          String key = tag.getKey();
-          if (key.charAt(0) != '@') {
-            keysToDelete.add(key);
-            tagsToAdd.add(tts.get().getOSMTagOf((OSHDBTag) tag.getValue()));
+        Map<String, Object> props = data.getProperties();
+        OSHDBTag[] tags = (OSHDBTag[]) props.remove("@tags");
+        if (tags != null) {
+          for (OSHDBTag tag : tags) {
+             OSMTag osmTag = tts.get().getOSMTagOf(tag);
+             String key = osmTag.getKey();
+             props.put((key.startsWith("@"))?"@"+key:key, osmTag.getValue());
```

Comment:
sorry to nag again about whitespace, but please take a look at the warnings from _checkstyle_, e.g. https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-36/8/checkstyle/new/category.-2024590691/source.62e6791e-ae76-48cc-b094-b0a6a77a47d3/#983 and https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-36/8/checkstyle/new/category.-641434199/source.93e95eb6-0bf9-4df4-b1ae-6beb57ee9e56/#983